nodeup/pkg/model/sysctls.go

```diff
@@ -148,6 +148,9 @@ func (b *SysctlBuilder) Build(c *fi.ModelBuilderContext) error {
 	sysctls = append(sysctls,
 		"# Prevent docker from changing iptables: https://github.com/kubernetes/kubernetes/issues/40182",
 		"net.ipv4.ip_forward=1",
+		"net.ipv4.ip_forward=1",
```

Comment:
I think these should be added conditionally when IPv6 is enabled. My guess is that they were disabled for security reasons on IPv4 only hosts.
Also, could we get this part as a separate PR, I would like to play with adding prefixes via launch templates?
```suggestion
		"net.ipv6.ip_forward=1",
```

Reply:
Nothing in this PR is of mergeable quality :) It just shows what I hacked together to get stuff working.

I think a PR with the sysctls can be done though. We can probably change accept_ra=2 on all interfaces rather than just the one.

Reply:
That would be great. Thanks.